src/plugins/data/common/query/persistable_state.ts

```diff
@@ -6,58 +6,53 @@
  * Side Public License, v 1.
```

Comment:
This is a new file, but a diff is broken because old file was moved:

src/plugins/data/common/query/persistable_state.ts -> src/plugins/data/common/query/filters/persistable_state.ts